operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -889,7 +902,7 @@ def generateAndValidate(self):
 
       # check if file name corresponds with config.xml or a module
 
-      if not self.moduleMap.has_key(the_file) and the_file != "config.xml":
+      if not self.moduleMap.has_key(the_file) and the_file != "config.xml" and the_file != "version.txt":
```

Comment:
Add a if the_file == "version.txt": continue right at line 895...    See my next comment for the reasoning.